README.md

```diff
@@ -1 +1,36 @@
-# BLS TypeScript Key Managment
+# BLS TypeScript Key Management
+
+![npm (tag)](https://img.shields.io/npm/v/bls-ts-key-mgmt/latest)
```

Comment:
Can we rename this to something simpler and more descriptive before we publish?
prefer something like `bls-eip2333`, `bls-hdwallet`, `bls-keygen` or something like that?

Reply:
I like bls-keygen, @GregTheGreek was naming this in first place :smile:

Reply:
ya wooops this looks better